<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>shortest_path</class>
 <widget class="QDialog" name="shortest_path">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="Task3Button">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Задание 3</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="AnswearEditT_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>201</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>37</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Ответ/Флойд-Уоршелл</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>41</y>
     <width>121</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Ответ/Дийкстра</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="AnswearEditT_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>121</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Task2Button">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>14</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Задание 2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="StartVertexEditT_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>62</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Task1Button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>16</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Задание 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ответ/Беллман</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="AnswearEditT_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>111</width>
     <height>244</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>43</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Стартовая вершина</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
